tests/kola/rpm-ostree/replace-rt-kernel

```diff
@@ -15,68 +18,51 @@ runv () {
 }
 
 basearch=$(arch)
-rhelver=$(. /usr/lib/os-release && echo "${RHEL_VERSION}")
-major=$(echo "${rhelver}" | cut -f 1 -d .)
+major=$(. /usr/lib/os-release && echo "${CPE_NAME}" | grep -Eo '[0-9]{1,2}')
 baseurl=
 case "${major}" in
-    8)
-        # TODO: why the heck does centos only support insecure http?!?
-        baseurl=http://mirror.centos.org/centos/8-stream/
-        # TODO avoid hardcoding versions
-        target_kver=4.18.0-383.el8
-        target_kver_rt=4.18.0-394.rt7.179.el8
-        ;;
     9)
-        # TODO: why the heck does centos only support insecure http?!?
-        baseurl=http://mirror.stream.centos.org/9-stream/
-        # TODO avoid hardcoding versions
-        target_kver=5.14.0-196.el9
-        target_kver_rt=5.14.0-201.rt14.202.el9
+        baseurl=https://mirror.stream.centos.org/9-stream/
+        repo_url=https://raw.githubusercontent.com/openshift/os/master/c9s.repo
         ;;
-    *)  fatal "Unhandled RHEL_VERSION=${rhelver}"
+    *)  fatal "Unhandled major RHEL/SCOS VERSION=${major}"
         ;;
 esac
 
+# setup repos
+runv rm -rf /etc/yum.repos.d/*
+runv curl -sSLf "${repo_url}" -o /etc/yum.repos.d/cs.repo
+runv curl -sSLf https://centos.org/keys/RPM-GPG-KEY-CentOS-Official -o /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-Official
+runv sed -i 's|^gpgkey.*|gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-Official|' /etc/yum.repos.d/cs.repo
```

Comment:
To me it looks like rpm-ostree may be failing when trying to verify the repos in c9s.repo. The keys for all of them have to be available. Blanket replacing all of them with one key won't work.

Reply:
hmm. but this test as written passes on RHCOS (@jmarrero, correct me if I am wrong).

According to @jmarrero it fails on SCOS though and we aren't sure why yet.

Reply:
And we do have `repo_gpgcheck=0` on the repos in c9s.repo. I might've pulled a false alarm here.

Reply:
Correct this passes on RHCOS both on CI & locally

Reply:
The problem here is that those reference `distribution-gpg-keys` which we don't ship in the OS by default.  I think @LorbusChris 's suggestion makes sense...

But I don't understand actually why we're removing the repo files by default?

Reply:
It is because this is a test for RHCOS only at the moment, I am just making sure there nothing on yum.repos.d before downloading the centos repo. When I get to the SCOS test, the plan is to just use the provided repos/keys and just turn on the rt repo if it's disabled.

Reply:
see also https://github.com/openshift/os/pull/1316